src/Models/Media.php

```diff
@@ -7,10 +7,17 @@
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Str;
 
+use HMTwill\Models\Behaviors\HasSchema;
+
 class Media extends Model
 {
+
+    use HasSchema;
```

Comment:
@raymondtri there seems to be remaining custom code that doesn't pertain to this PR.